Platforms/QemuQ35Pkg/Plugins/QemuRunner/QemuRunner.py

```diff
@@ -227,11 +227,14 @@ def Runner(env):
         if monitor_port is not None:

```

Comment:
Do these changes need to be mirrored to QemuSbsa's QemuRunner? At a minimum it seems like the `stty sane` would need to be done there too.